pkg/sdkclient/serverinfo/types.go

```diff
@@ -70,14 +86,54 @@ A constraint ">=0.8.0-z" will match any pre-release version of 0.8.0, including
 More details about version comparison can be found in the PEP 440 and semver documentation.
 */
 var minimumSupportedSDKVersions = sdkConstraints{
```

Comment:
we need to figure out how we can read this from a YAML file. maintaining the code is tricky.

Reply:
maybe we can use the go embed. we can revisit at a later time.